pkg/configuration/configuration.go

```diff
@@ -145,23 +145,51 @@ const (
 	DefaultVerificationCodeExpiresInMin = 5
 )
 
+type Configuration interface {
```

Comment:
The reason for introducing this interface was twofold; firstly it simplified some aspects of the tests, but mainly I was thinking ahead to the enhancements we intend to make for configuration in general.  If we wish to introduce the planned enhancements for configuration (loading config from CRDs etc) then we will need a new implementation, and by that stage it starts to make more sense in my estimation for the consumers of the configuration to work off an interface.

If we were to simply override the environment variables at the beginning of the tests, then wouldn't that need to be rewritten once the configuration was refactored to be based on CRDs?

Reply:
Ok, I didn't think about the CRDs usage. That's a good point. 
We will see what it will look like and if we will query the `HostOperatorConfig` CR from the registration service or if we will still use the `ConfigMap` for this particular deployment. 
We will need to change the code in host-operator deployment anyway and make the usage of the new CR consistent in both repos (host-operator & reg-service), so I was thinking that if we had the code similar now, then the future changes would be easier.

Let's keep it as it is - the code is already done, working and looks fine - we can change it later when we start using the CRD